ui/cryptomaterial/icon_gallery.go

```diff
@@ -202,6 +202,7 @@ func (i *Icons) DefaultIcons() *Icons {
 	i.MultiWalletIcon = NewImage(decredIcons["multiWalletIcon"])
 	i.IntegratedExchangeIcon = NewImage(decredIcons["integratedExchangeIcon"])
 	i.CrossPlatformIcon = NewImage(decredIcons["crossPlatformIcon"])
+	i.BlueAddIcon = NewImage(decredIcons["add_blue_icon"])
```

Comment:
```suggestion
	i.AddIcon = NewImage(decredIcons["add_blue_icon"])
```
The icon colour should not be inferred in the name, one icon should be sufficient, and if needed can be changed programatically